client/src/components/ratings/sub-components/ReviewDashboard.jsx

```diff
@@ -1,4 +1,5 @@
 import React, {useState, useEffect} from 'react';
+import Star from '../images/star.png';
```

Comment:
```suggestion

```

Reply:
prompting removal since it was only for demonstrative purposes, don't want any surprises for erik